apps/renterd/contexts/config/transform.spec.ts

```diff
@@ -526,29 +518,26 @@ describe('tansforms', () => {
 
 function buildAllResponses() {
   return {
-    autopilotInfo: {
-      state: {
-        id: 'autopilot',
-        configured: true,
-        migrating: true,
-        migratingLastStart: new Date().toISOString(),
-        scanning: true,
-        scanningLastStart: new Date().toISOString(),
-        synced: true,
-        uptimeMS: '333',
-        network: 'mainnet' as const,
-        version: '0.0.0',
-        commit: 'commit',
-        OS: 'os',
-        buildTime: new Date().getTime(),
-        explorer: {
-          enabled: true,
-          url: 'https://api.siascan.com',
-        },
-        startTime: new Date().getTime(),
+    autopilotState: {
+      id: 'autopilot',
+      configured: true,
```

Comment:
This was updated to `enabled`. I imagine this'll be updated again soon and maybe even removed (for the most part).

Reply:
This PR adds `enabled` to the autopilot config: https://github.com/SiaFoundation/web/pull/822 

But isn't there still `configured` in terms of autopilot state? I still make use of configured throughout the code to understand if the user has done initial setup. If its gone does this mean there are default values now?

Reply:
Re `configured`: https://github.com/SiaFoundation/renterd/blob/b98cc3d965700f7b675931fb8688e696bfeb1027/api/autopilot.go#L88